examples/requirements-and-tests/README.md

```diff
@@ -0,0 +1,71 @@
+# Multi Language Example (Requirements Model and Test Model)
+
+## Overview
+
+This example allow to model Requirement and Tests Cases in two separate models (with separate languages). Test Cases can reference Requirements (cross language references).
+
+We model Requirements in a simple Requirement Model (Requirements consist of and identifier and a requirement text):
+
+```text
+req ReqId001 "The cli-tool should allow to extract all test cases referencing a specific requirement"
+req R1_tstID "A test identifier shall contain a number."
+req RA_reqID "A requirement identifier shall contain a number."
+req R3_reqCov "A requirement shall be covered by at least one test"
+```
+
+Test Cases have also an identifier and the references to the Requirements checked by the test.
+
+## What does this example demonstrate
+
+* How to integrate two languages to interact with each other: They use common rules (e.g., `ID` or `Contact`) and they have cross languages references (e.g. `Requirement` or `Environment`).
+* How to setup these languages with their services and grammars (file layout).
+* How to create cross language validations (are requirements covered by some tests?).
+* How to create a cross language generator (coverage matrix).
+
+## Language Design
+
+Grammars:
+
+* `common.langium`: common aspects used the main grammars.
+* `requirements.langium`: requirements language (includes `common.langium`)
+* `tests.langium`: requirements language (includes `common.langium` and `requirements.langium`, references `Requirement` elements from `requirements.langium`).
+
+Modules and Services:
+
+* `createRequirementsAndTestsLanguageServices` creates the full set of services used by the CLI and the language server.
+* `requirements-language-module.ts`: requirements service configuration
+* `tests-language-module.ts`: tests service configuration
+
+Language configuration:
+
+* `langium-config.json`: TODO, what does this exactly (AST code generation, syntax highlighting, TextMate).
+* `package.json`: `contributes` section, TODO, what does this exactly (vscode extension linkage?).
```

Comment:
@spoenemann Can we clarify this?

Reply:
You can point to https://code.visualstudio.com/api/references/contribution-points#contributes.languages for documentation.